asv_bench/benchmarks/groupby.py

```diff
@@ -90,45 +93,6 @@ def time_series_groups(self, data, key):
         self.ser.groupby(self.ser).groups
 
 
-class FirstLast(object):
```

Comment:
I removed this class because it is duplicative of other tests. `first` and `last` are already covered in the `GroupByMethods` class. For the `nth` tests (although they are arguably duplicated of `first`) I ended up moving them all to one shared `Nth` class that covers the various dtype permutations